packages/base-styles/_z-index.scss

```diff
@@ -31,12 +31,14 @@ $z-layers: (
 	".interface-interface-skeleton__header": 30,
 	".interface-interface-skeleton__content": 20,
 	".edit-widgets-header": 30,
-	".wp-block-cover__inner-container": 1, // InnerBlocks area inside cover image block.
+
 	".wp-block-cover.is-placeholder .components-placeholder.is-large": 1, // Cover block resizer component inside a large placeholder.
+	// These z-index are now used only for a deprecated version of the cover block.
+	".wp-block-cover__inner-container": 1, // InnerBlocks area inside cover image block.
 	".wp-block-cover.has-background-dim::before": 1, // Overlay area inside block cover need to be higher than the video background.
-	".block-library-cover__padding-visualizer": 2, // BoxControl visualizer needs to be +1 higher than .wp-block-cover.has-background-dim::before
```

Comment:
I'm removing it again because the change discussed [here](https://github.com/WordPress/gutenberg/pull/65626#discussion_r1775333362) was [reverted](https://github.com/WordPress/gutenberg/pull/66074).